tests/test_integration/test_settings.py.sample

```diff
@@ -0,0 +1,2 @@
+url = 'https://wl-test.example.com/'
```

Comment:
I like that. And it answers the question I asked in https://github.com/yola/yolapy/pull/26#discussion_r44524450

But what do you think about changing this in a subsequent pull that we can prioritize after some other issues that we're falling behind on?